src/frontend/apps/markdown/MdxRenderer/constants.tsx

```diff
@@ -0,0 +1 @@
+export const debouncingTime = 1000; // wait in milliseconds before rendering
```

Comment:
you can rename this file extension in `.ts`, you are not using jsx in this component